packages/core/src/index.ts

```diff
@@ -42,6 +42,7 @@ export type {
 } from './types'
 
 export type { EIP1193Provider } from '@web3-onboard/common'
+export { connectWallet$ } from './streams'
```

Comment:
Is this necessary?

Reply:
You are right, It's not necessary. Removed.

Reply:
@Adamj1232 fixed all comments. Pls, review again.